packages/governance/src/contractHelper.js

```diff
@@ -5,31 +5,30 @@ import { keyEQ } from '@agoric/store';
 // eslint-disable-next-line -- https://github.com/Agoric/agoric-sdk/pull/4837
 import { CONTRACT_ELECTORATE } from './paramGovernance/governParam.js';
 import { assertElectorateMatches } from './paramGovernance/paramManager.js';
+import { makeParamManagerFromTerms } from './paramGovernance/typedParamManager.js';
 
 const { details: X, quote: q } = assert;
 
 /**
  * Helper for the 90% of contracts that will have only a single set of
  * parameters. In order to support managed parameters, a contract only has to
  *   - define the parameter template, which includes name, type and value
- *   - call handleParamGovernance() to get wrapPublicFacet and wrapCreatorFacet
+ *   - call facetHelpers() to get augmentPublicFacet and makeGovernorFacet
```

Comment:
If I understand the point here, this comment needs to be updated to distinguish it from the (currently identical) comment on the new `handleParamGovernance`.  It appears that that one is for the 90% case, and this is for something else, though I'm not clear what.   I think `facetHelpers` also returns `params`, but it requires that the `paramManager` be provided explicitly. 

What's the model for when facetHelpers would be useful?

Reply:
I can't think of a reason why a contract would want to use `facetHelpers`.  The two cases I can think of are 1) satisfied by the new `handleParamGovenance`, and 2) doing things manually.  I would certainly not document `facetHelpers` as "for the 90% case, and I think it would be better to not export it until/unless we find a reason to want to use it.

Reply:
Ok cool, I've removed it from the exports and comments 31a7d7d63940f62747c02767e7be08a2e5883183